alphadia/workflow/peptidecentric.py

```diff
@@ -521,24 +370,17 @@ def first_recalibration_and_optimization(
         fragments_df : pd.DataFrame
             Fragment dataframe from optimization lock
 
-        ordered_optimizers : list
-            List of lists of optimizers in correct order
         """
+
+        self.optimization_manager.fit(
+            {"classifier_version": self.fdr_manager.current_version}
+        )
         precursor_df_filtered, fragments_df_filtered = self.filter_dfs(
             precursor_df, fragments_df
         )
 
         self.recalibration(precursor_df_filtered, fragments_df_filtered)
 
-        self.reporter.log_string(
-            "=== Performing initial optimization on extracted data. ===",
-            verbosity="info",
-        )
-
-        for optimizers in ordered_optimizers:
-            for optimizer in optimizers:
-                optimizer.step(precursor_df_filtered, fragments_df_filtered)
-
     def calibration(self):
```

Comment:
should we rename this to `search_parameter_optimization`?

Reply:
Or just `optimization`?

Reply:
would be less specific (there might be other optimizations?), but also fine ..

Reply:
I suppose so; let's go for the longer name then.